frontend/src/scenes/plugins/edit/PluginDrawer.tsx

```diff
@@ -52,6 +53,24 @@ export function PluginDrawer(): JSX.Element {
         }
     }, [editingPlugin?.id])
 
+    const isValidChoiceConfig = (configField: PluginConfigChoice): boolean => {
+        /*
+            Checks, in order, that `choices`:
+            1. Exists   
+            2. Is an array or object
+            3. Is not an object 
+            4. Is not an empty array
+            5. Is an array of strings only
+        */
+        const invalid =
+            !configField.choices ||
+            typeof configField.choices !== 'object' ||
+            JSON.stringify(configField.choices)[0] === '{' ||
```

Comment:
many other more elegant solutions weren't actually working here at runtime

Reply:
I've found `Array.isArray` to work quite nicely. Won't something like `Array.isArray(configField.choices) && !configField.choices.find(c => typeof c !== 'string')` do the trick here?

Reply:
This is good: `!configField.choices.find(c => typeof c !== 'string')` but I didn't manage to get `Array.isArray` working earlier - could have been a cache/hot reloading issue, will try again